osbuild/util/jsoncomm.py

```diff
@@ -299,7 +299,7 @@ def send(self, payload: object, *, destination: Optional[str] = None, fds: list
 
         serialized = json.dumps(payload).encode()
         cmsg = []
-        if len(fds) > 0:
+        if fds and len(fds) > 0:
```

Comment:
Couldn't that be just `if fds:` as the truthiness of an empty list is `False`?

Reply:
Right! Empty arrays are true in Javascript, but false in python. I always mess this up. I have a follow-up locally, since I have more reworks of the tests ;)